plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/upload-plugin.spec.js

```diff
@@ -14,15 +14,22 @@ const {
 	installPluginThruWpCli,
 } = require( '../../utils/plugin-utils' );
 
+const skipMessage = 'Skipping this test because PLUGIN_REPOSITORY is undefined';
+
 let pluginPath;
 let pluginSlug;
 
-test.describe( `${ PLUGIN_NAME } plugin can be uploaded and activated`, () => {
-	test.skip(
-		! PLUGIN_REPOSITORY,
-		`SKIPPED: PLUGIN_REPOSITORY is falsy: ${ PLUGIN_REPOSITORY }`
-	);
+test.skip( () => {
+	const shouldSkip = ! PLUGIN_REPOSITORY;
+
+	if ( shouldSkip ) {
+		console.log( skipMessage );
+	}
 
+	return shouldSkip;
+}, skipMessage );
+
+test.describe( `${ PLUGIN_NAME } plugin can be uploaded and activated`, () => {
```

Comment:
Same reasoning in https://github.com/woocommerce/woocommerce/pull/35873/files#r1042185497